### ISSUE TITLE
Moved files aren't transplanted

### ISSUE THREAD

**Issue by jdm** [servo-automation/upstream-wpt-sync-webhook#5]
https://github.com/servo/servo/pull/19979
https://github.com/jdm/web-platform-tests/pull/26/

**Comment by jdm** [comment]
Filterdiff's output is incorrect:
```diff
diff --git a/tests/wpt/web-platform-tests/2dcontext/transformations/2d.transformation.scale.large.html b/tests/wpt/web-platform-tests/2dcontext/transformations/2d.transformation.scale.large.html
deleted file mode 100644
index 926530d1f7e6..000000000000
+++ /dev/null
@@ -1,33 +0,0 @@
-<!DOCTYPE html>
-<!-- DO NOT EDIT! This test has been generated by tools/gentest.py. -->
-<title>Canvas test: 2d.transformation.scale.large</title>
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
-<script src="/common/canvas-tests.js"></script>
-<link rel="stylesheet" href="/common/canvas-tests.css">
-<body class="show_output">
-
-<h1>2d.transformation.scale.large</h1>
-<p class="desc">scale() with large scale factors works</p>
-
-<p class="notes">Not really that large at all, but it hits the limits in Firefox.
-<p class="output">Actual output:</p>
-<canvas id="c" class="output" width="100" height="50"><p class="fallback">FAIL (fallback content)</p></canvas>
-<p class="output expectedtext">Expected output:<p><img src="/images/green-100x50.png" class="output expected" id="expected" alt="">
-<ul id="d"></ul>
-<script>
-var t = async_test("scale() with large scale factors works");
-_addTest(function(canvas, ctx) {
-
-ctx.fillStyle = '#f00';
-ctx.fillRect(0, 0, 100, 50);
-
-ctx.scale(1e5, 1e5);
-ctx.fillStyle = '#0f0';
-ctx.fillRect(0, 0, 1, 1);
-_assertPixel(canvas, 50,25, 0,255,0,255, "50,25", "0,255,0,255");
-
-
-});
-</script>
-
```

**Comment by jdm** [comment]
Filed https://github.com/twaugh/patchutils/issues/22.

**Comment by jdm** [comment]
This is fixed by a29f22d98874fe73084a18d5b9a64a4917c4b6c4.
